orbbec_acquire/util.py

```diff
@@ -272,15 +274,15 @@ def start_recording(
     system_timestamps = []
     device_timestamps = []
     start_time = time.time()
-    print(start_time)
+    print("Start time:", start_time)
     count = 0
 
     # the actual recording
     try:
         while time.time() - start_time < recording_length:
-            frames = pipeline.wait_for_frames(1000)
+            frames = pipeline.wait_for_frames(100)
             if frames is None:
-                print("Dropped frame")
```

Comment:
I prefer to keep this.

Reply:
What does the `wait_for_frames` function expect? Is 1000 a unit of time in milliseconds or something? I assumed it was something like that so I reduced the wait time because I noticed on the system I tested with, the recording rate was 25Hz instead of 30. Do you know if setting this number too high will reduce the frame rate?

Reply:
https://github.com/orbbec/pyorbbecsdk/blob/main/docs/README_EN.md#video-stream-acquisition

This function is described here. It is the time_out time to wait for all the frames from both ir and depth stream, this is 1000 ms, and I don't think changing this number changes frame rate. Frame rate can be explicitly set in this function ` profile_list.get_video_stream_profile`